src/data/battle_moves.h

```diff
@@ -1821,7 +1821,7 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_Z] =
     [MOVE_MIMIC] =
     {
         .effect = EFFECT_MIMIC,
-        .power = 0,
```

Comment:
Because of the way Mimic works I think its base power should stay 0 - Mimic itself never does damage.